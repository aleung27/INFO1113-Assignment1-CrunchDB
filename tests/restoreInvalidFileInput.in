RESTORE restoreInvalid.txt
LIST KEYS
LIST ENTRIES
LIST SNAPSHOTS
BYE
