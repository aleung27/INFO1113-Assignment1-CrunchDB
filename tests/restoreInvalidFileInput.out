> invalid input

> no keys

> no entries

> no snapshots

> bye
